config/product/explorer.js

```diff
@@ -211,7 +217,18 @@ export function init(store) {
   });
 
   virtualType({
-    label:          'Overview',
+    label:       store.getters['i18n/t']('members.clusterMembers'),
+    group:      'rbac',
```

Comment:
I got in a pickle with this location by incorrectly assuming roles created in this RBAC section's `ClusterRoles` page should then apply to the roles shown when in `Cluster Members`. Minus membership all other sections are core kube `rbac.authorization.k8s.io` resource types. Feels like  `Cluster Membership` should move into the top level `Cluster` section alongside projects + nodes. Ideally everything role/membership related should be in RBAC but I don't think we can escape from the management cluster vs rbac cluster confusion

Reply:
I mean I put this here because as you said ideally everything role/membership related should be in RBAC. I'd prefer to organize this by concept and not by types. I'm going to leave this as is for now but if it does lead to more confusion I'll go ahead and change it.

Reply:
I'd rather keep this as is. As you mentioned ideally it's in the right spot. If it does add more confusion then I'll change it.